forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/CrmDelegate.java

```diff
@@ -117,7 +121,7 @@ private void crmOperation(DelegateExecution execution) {
         }
 
         // Create a new incident in CRM
-        CrmPostResponse crmPostResponse = createCrmIncident(contactId);
+        CrmPostResponse crmPostResponse = createCrmIncident(contactId, threadText);
```

Comment:
What do you think of passing the `execution` object to the `createCrmIncident` method and then load form properties in that method? We'll need to load several properties in that method including `subject` which I'm currently working on. This way we won't need to pass several parameters to `createCrmIncident`, the method responsible for loading relying on `execution` object.

Reply:
I was suppose to discuss the same with you. As Even in thread we have possible 3 params that we might start using in future let me fix that.